### PR TITLE
Update ID devs

### PR DESCRIPTION
**Pull request by muslih**
Update id devs number based on Organization chart Nov 1st. 2018

### CODE REVIEW

**Comment by muslih** [comment]
Please review @quipper/video-id-devs

File: company-profile-en.md [hunk 1]
```diff
@@ -20,9 +20,9 @@
 | --- | --- | --- | --- | --- | --- |
 | Tokyo | 18 | 7 | 3 | 4 | 32 |
 | Manila | 6 | 2 | 1 | 0 | 9 |
-| Jakarta | 4 | 2 | 0 | 3 | 9 |
+| Jakarta | 7 | 4 | 0 | 3 | 9 |
```
Comment (irfanfadilah) [inline company-profile-en.md]: The total of the product team (rightmost) should be updated accordingly too, 7+4+3=14.

Comment (fudanchii) [reply company-profile-en.md]: @irfanfadilah right, nice catch, I didn't do any calculation before :sweat:, could you open pull request to fix it?

thanks! :dancer:

Comment (irfanfadilah) [reply company-profile-en.md]: @fudanchii added: https://github.com/quipper/handbook/pull/36 👍

**Pull request merged**